BraintreeCore/src/test/java/com/braintreepayments/api/core/BraintreeClientUnitTest.kt

```diff
@@ -377,19 +377,24 @@ class BraintreeClientUnitTest {
         val sut = createBraintreeClient(configurationLoader)
         sut.reportCrash()
 
-        val callbackSlot = slot<ConfigurationLoaderCallback>()
+        val callbackSlots = mutableListOf<ConfigurationLoaderCallback>()
         verify {
-            configurationLoader.loadConfiguration(capture(callbackSlot))
+            configurationLoader.loadConfiguration(capture(callbackSlots))
         }
 
-        callbackSlot.captured.onResult(ConfigurationLoaderResult.Success(configuration))
+        callbackSlots[0].onResult(ConfigurationLoaderResult.Success(configuration))
```

Comment:
Do we want to consider only ever having 1 configuration? This would likely be a seperate PR, but we could consider something similar to [this logic on iOS](https://github.com/braintree/braintree_ios/blob/6582191edb667a33a5e5900421b27edf3db2f88e/Sources/BraintreeCore/Configuration/ConfigurationLoader.swift#L47-L50) where we only ever return 1 config result even if there are multiple in flight at the same time on slower networks.

Reply:
Yep! I think that's what we'll want to move towards. Sai and I talked about replacing the configuration loader callbacks with Coroutine Flow. This way all parallel request to configuration will rely on a single API call or configuration stored in cache.

This would be a large change to the SDK so we decided to tackle it after as a tech debt item.

Reply:
Awesome! Yeah definitely outside of the scope for this PR.